test/TesterInternal/General/ConsistentRingProviderTests_Silo.cs

```diff
@@ -172,7 +169,7 @@ public override TestingSiloHost CreateSiloHost()
             this.HostedCluster.StartAdditionalSilos(numAdditionalSilos);
             await this.HostedCluster.WaitForLivenessToStabilizeAsync();
             //List<SiloHandle> failures = getSilosToFail(Fail.Random, 1);
-            SiloHandle fail = this.HostedCluster.Secondary;
+            SiloHandle fail = this.HostedCluster.GetActiveSilos().Skip(1).First();
```

Comment:
agree, I will change that.    I wasn't clear if the term "secondary" was going away.